src/main/java/frc/robot/commands/auto/PathPlannerCommand.java

```diff
@@ -0,0 +1,66 @@
+package frc.robot.commands.auto;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import com.pathplanner.lib.PathConstraints;
+import com.pathplanner.lib.PathPlanner;
+import com.pathplanner.lib.PathPlannerTrajectory;
+import com.pathplanner.lib.PathPoint;
+import com.pathplanner.lib.commands.PPSwerveControllerCommand;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import edu.wpi.first.wpilibj2.command.PrintCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.DoNothing;
+import frc.robot.constants.AutoConstants;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.PathGroupLoader;
+
+
+// Assuming this method is part of a drivetrain subsystem that provides the necessary methods
```

Comment:
This is left over from copy pasting it from the documentation. remove it  or update it to make it clearer in the context. maybe even link to the documentation.